proto/reporting/input_params/front_desk_report_manifests.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+package holms.types.reporting.input_params;
+option csharp_namespace = "HOLMS.Types.Reporting.ReportParams";
+option go_package = "reporting/input_params";
+
+import "google/protobuf/timestamp.proto";
+import "primitive/pb_inclusive_opsdate_range.proto";
+import "primitive/pb_local_date.proto";
+
+enum InHouseGuestReportSorting {
+	BY_GUEST_NAME = 0;
+	BY_ROOM_NUMBER = 1;
+}
+
+message FrontDeskCalendarDateReportManifest {
+	google.protobuf.Timestamp date = 1;
```

Comment:
Note: see comment below re: filtering remark. I see that you put it on the top-level request object.

It might make sense to change the UI to reflect the top-level property selection, since I don't see a case where we'd ever report on two different properties in the same batch request.